source/include/platform/acmacosx.h

```diff
@@ -119,6 +119,7 @@
 #include "aclinux.h"
 
 #ifdef __APPLE__
+#include <semaphore.h>
 #define sem_destroy         sem_close
```

Comment:
What if in osunixxf.c:

#ifdef __APPLE__
#include <semaphore.h>
#endif

#include other files